.github/workflows/build-os-image.yml

```
@@ -314,9 +318,9 @@ jobs:
       - name: Download VMGS blob
         run: |
           aws s3 cp \
-            --region ${AZURE_VMGS_REGION} \
-            s3://constellation-secure-boot/${PKI_SET}/${AZURE_SECURITY_TYPE}.vmgs \
-            ${PKI_SET}/${AZURE_SECURITY_TYPE}.vmgs \
+            --region "${AZURE_VMGS_REGION}" \
+            s3://constellation-secure-boot/"${PKI_SET}"/"${AZURE_SECURITY_TYPE}".vmgs \
+            "${PKI_SET}"/"${AZURE_SECURITY_TYPE}".vmgs \
```

```suggestion
            --region "${AZURE_VMGS_REGION}" \
            "s3://constellation-secure-boot/${PKI_SET}/${AZURE_SECURITY_TYPE}.vmgs" \
            "${PKI_SET}/${AZURE_SECURITY_TYPE}.vmgs" \
```

nit. IMO easier to read.